scripts/lib/install_node.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# Install Node and necessary frameworks/packages
+
+# Install Node.js on Linux as root on RHEL, CentOS, CloudLinux, or Fedora
+install_node() {
+  if which node > /dev/null
```

Comment:
Adopt the `command -v ` form seen in other install scripts to check if program exists